### ISSUE TITLE
Reviewer 1 Comment 7

### ISSUE THREAD

**Issue by ramess101** [ramess101/JCED_FOMMS_Manuscript#25]
@mrshirts @jpotoff @msoroush 

> In section 3.3, the authors show the effective snapshot results for GCMC-MBAR and use them to explain the bad performance of the method in predicting liquid phase properties. They also compare with MBAR-ITIC but the comparison is not very clear to me. It would be nice to have some data from MBAR-ITIC listed in the comparison. Also the authors listed two of their hypothesis that GCMC-MBAR would experience better overlap than MBAR-ITIC when 𝜃𝑟𝑟 ≉ 𝜃𝑟𝑒𝑓. Are these hypothesis proved to be correct after the comparison? Does GCMC-MBAR still does relatively better even for liquid phase since the two hypothesis the authors raise should hold true for liquid phase.

Any ideas for what the reviewer intended with the last sentence? 

I think there is some confusion regarding MBAR-ITIC, which is understandable since it is not a common method and we only reference it briefly. The whole reason we mention MBAR-ITIC here is to point out that MBAR-ITIC fails miserably when lambda_rr \neq lambda_ref, while GCMC-MBAR is still relatively reliable. But there really is no way to compare MBAR-ITIC and GCMC-MBAR directly because the state points are completely different. 

However, I do agree that we need to make it clear that these results support our hypothesis, namely, GCMC-MBAR yields more reliable estimates with greater Keff than MBAR-ITIC for theta_rr \not\approx theta_ref.

**Comment by mrshirts** [comment]
Yeah, I think this has to be clearer what.  MBARR-GCMC doesn't perform poorly, it just doesn't perform as well as it does in liquid.  You can't actually do what it does with another method (AFAIK).  So it's just a limitation on the method, not a drawback.

**Comment by ramess101** [comment]
@mrshirts @jpotoff @msoroush 

This comment actually raises a slightly more complicated issue than I originally thought. This is the paragraph regarding our initial hypothesis:

![image](https://user-images.githubusercontent.com/23408516/53758601-7537c700-3e7b-11e9-8530-b6b8bfcf4a42.png)
![image](https://user-images.githubusercontent.com/23408516/53758612-7b2da800-3e7b-11e9-8470-0bb438890a0a.png)

I feel like our initial hypothesis is somewhat misleading because we can't really compare the overlap of MBAR-ITIC and GCMC-MBAR. In both cases we can quantify overlap with Keff for a different theta than theta_ref. However, MBAR-ITIC Keff is for NVT simulations while GCMC-MBAR is for GC simulations. Furthermore, the state points are completely different: MBAR-ITIC performs NVT simulations along the supercritical temperature (ranging from vapor to liquid like densities) and along compressed liquid isochores. By contrast, GCMC-MBAR performs GC simulations at mu and T conditions that correspond to compressed liquid, near critical point fluid, and rarefied vapor. 

The one comparison we can make is how well MBAR-ITIC or GCMC-MBAR predicts saturated vapor or liquid properties for large changes in theta. This comparison clearly demonstrates that GCMC-MBAR is superior. But this is related to the fact that MBAR-ITIC requires reasonable pressure estimates of the liquid phase, which require a large Keff in the compressed liquid isochore state points.

Also, we store two orders of magnitude more configurations in our GCMC-MBAR analysis than in the MBAR-ITIC study.

**Comment by ramess101** [comment]
@mrshirts @jpotoff @msoroush 

After thinking about this some more, I guess the best comparison we can make between MBAR-ITIC and GCMC-MBAR is the Keff values at or near saturated liquid. For MBAR-ITIC this corresponds to the lowest temperature on the liquid density isochore, which is simulated directly. For GCMC-MBAR it corresponds to the saturated liquid determined by equating the pressures in the vapor and liquid phases. 

The scenario of interest is predicting MiPPE (lam = 16) from configurations sampled with TraPPE (lam = 12). Our previous MBAR-ITIC study reported that Keff is around 1 (i.e., only a single configuration contributes to the average) for the near saturated liquid. By contrast, this study shows that GCMC-MBAR always has a Keff > 1. However, the GCMC-MBAR Keff is still less than 50 for TraPPE -> MiPPE for saturated liquid. Also, GCMC-MBAR starts with about two orders of magnitude more configurations. Although it might seem nice to compare a percent Keff, you cannot really make a percentage Keff comparison because the Keff of 1 is a lower bound, i.e., it is not possible to have Keff < 1. 

I think to satisfy this reviewer we just need to state that MBAR-ITIC would have Keff \approx 1 for the saturated liquid state points, while GCMC-MBAR always has Keff > 1.

For the second part of the question, even when GCMC-MBAR has less than 50 in the liquid phase the results for the other properties are still reliable enough that a meaningful optimization is possible for changing lambda.

**Comment by ramess101** [comment]
@mrshirts @jpotoff @msoroush 

OK, I think I have made a valid comparison between MBAR-ITIC and GCMC-MBAR, which should satisfy the reviewer and help the reader:

![image](https://user-images.githubusercontent.com/23408516/53907571-b4e0e900-400a-11e9-95e0-ad14a3963235.png)

![image](https://user-images.githubusercontent.com/23408516/53907593-bca08d80-400a-11e9-8f57-e679e1ea58b4.png)

![image](https://user-images.githubusercontent.com/23408516/53907607-c1fdd800-400a-11e9-82f8-7ad437bff071.png)

**Comment by ramess101** [comment]
@mrshirts 

Could you look over my response and manuscript additions regarding GCMC-MBAR compared with MBAR-ITIC?

![image](https://user-images.githubusercontent.com/23408516/53976241-deaa1680-40c3-11e9-9673-8e62fd898713.png)
![image](https://user-images.githubusercontent.com/23408516/53976255-e5d12480-40c3-11e9-9631-5265c533b2c9.png)
